tests/src/e2e.spec.ts

```diff
@@ -54,34 +98,92 @@ test.skip('Check welcome page is displayed and that we are redirected to the mai
+    expect(telemetryConsent).not.undefined;
+    expect(await telemetryConsent.isChecked()).to.be.true;
 
-  // click on the button
-  await goToPodmanDesktopButton.click();
+    await telemetryConsent.click();
+    expect(await telemetryConsent.isChecked()).to.be.false;
+  });
 
-  await page.screenshot({
-    path: 'tests/output/screenshots/screenshot-welcome-page-redirect-to-dashboard.png',
-    fullPage: true,
+  test('Redirection from Welcome page to Dashboard works', async () => {
+    const goToPodmanDesktopButton = page.locator('button:text("Go to Podman Desktop")');
+    // wait for visibility
+    await goToPodmanDesktopButton.waitFor({ state: 'visible' });
+
+    await page.screenshot({ path: 'tests/output/screenshots/screenshot-welcome-page-display.png', fullPage: true });
+
+    // click on the button
+    await goToPodmanDesktopButton.click();
+
+    await page.screenshot({
+      path: 'tests/output/screenshots/screenshot-welcome-page-redirect-to-dashboard.png',
+      fullPage: true,
+    });
+
+    // check we have the dashboard page
+    const dashboardTitle = page.getByRole('heading', { name: 'Dashboard' });
+    await playExpect(dashboardTitle).toBeVisible();
+  });
+
+  test('Verify main UI elements are present in Status Bar', async () => {
+    await playExpect.soft(page.locator('xpath=.//li//div[@title="Help"]'), 'Help was not found').toBeVisible();
+    await playExpect.soft(page.locator('xpath=.//li//div[@title="Tasks"]'), 'Help was not found').toBeVisible();
+    await playExpect
+      .soft(page.locator('xpath=.//li//div[@title="Share your feedback"]'), 'Share your feedback was not found')
+      .toBeVisible();
+    await playExpect
+      .soft(page.locator('xpath=.//li//div[starts-with(@title, "Using version")]'), 'Version was not found')
+      .toBeVisible();
   });
 
-  // check we have the dashboard page
-  const dashboardTitle = page.getByRole('heading', { name: 'Dashboard' });
-  await playExpect(dashboardTitle).toBeVisible();
+  test.skip('Verify main UI elements are present in Status Bar - Kind and Compose installation', async () => {
+    if (existsSync(kindPath)) {
+      console.log('Kind is already downloaded');
+    } else if (binaryOnPath('kind')) {
+      console.log('Kind is on PATH');
+    } else {
+      await playExpect
+        .soft(page.getByRole('listitem').filter({ hasText: 'Kind' }), 'Kind button was not found')
+        .toBeVisible();
+    }
+    await playExpect
+      .soft(page.getByRole('listitem').filter({ hasText: 'Compose' }), 'Compose button was not found')
+      .toBeVisible();
+  });
 
-  expect(windowState.isCrashed, 'The app has crashed').toBeFalsy();
-  expect(windowState.isVisible, 'The main window was not visible').toBeTruthy();
+  test('Verify main UI elements are present in Navigation Bar', async () => {
+    for (const item of navBarItems) {
+      const locator = page.getByRole('navigation').getByText(new RegExp(item));
```

Comment:
we have navbarItems being
```
const navBarItems = ['Dashboard', 'Containers', 'Images', 'Pods', 'Volumes', 'Settings'];
```

so I'm not sure why we're using a Regexp ? `getByText(new RegExp(item));`

I would also expect that it checks that we have clickable (aria-role) elements, not just the text

Reply:
> we have navbarItems being
> 
> ```
> const navBarItems = ['Dashboard', 'Containers', 'Images', 'Pods', 'Volumes', 'Settings'];
> ```
> 
> so I'm not sure why we're using a Regexp ? `getByText(new RegExp(item));`

Because text changes based on amount of items in the concrete page. Images (5), etc...

> I would also expect that it checks that we have clickable (aria-role) elements, not just the text
Absolutely.